### PR TITLE
Nmaynard/legacy ids

### PR DESCRIPTION
**Pull request by nigel-smk**
This adds the legacy_id column to the hostGenomes and the workflow_runs tables.

See the linkml edits commit for the changes that I made. The rest of the commits are codegen/migrations.

There are some old linkml changes that didn't have their associated migration changes commit in the workflows previously that this alembic migration generation picked up. I guess it's not so bad to have this PR do that catchup as well?

### CODE REVIEW

**Review by morsecodist — APPROVED** [review]

File: entities/docker-compose.yml [hunk 1]
```diff
@@ -14,6 +14,8 @@ services:
       # directory as its storage location instead so it works with our db snapshot
       # workflow.
       - PGDATA=/var/lib/platformics/data
+    ports:
```
Comment (robertaboukhalil) [inline entities/docker-compose.yml:17]: Is this change needed?

**Review by robertaboukhalil — APPROVED** [review]
Looks good, though can you undo the change of ports in docker-compose.yml or is that needed? Migrations look good. You'll need to run `make fix-lint` in both entities/ and workflows/ to fix the lint errors

**Comment by nigel-smk** [comment]
> Looks good, though can you undo the change of ports in docker-compose.yml or is that needed? Migrations look good. You'll need to run `make fix-lint` in both entities/ and workflows/ to fix the lint errors

I meant to leave a note about exposing the port. The workflows db exposes a port and I have been exposing the port on entities db to be able to connect from a db manager (e.g. pgadmin). I figured I would commit it so that it's easier for others to do the same.

**Comment by robertaboukhalil** [comment]
Ah I see, got it
